src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/service/AsyncActionPlanEventService.kt

```diff
@@ -20,15 +20,15 @@ class AsyncActionPlanEventService(
   private val timelineEventFactory: TimelineEventFactory,
   private val timelineService: TimelineService,
 ) : ActionPlanEventService {
-  override fun actionPlanCreated(actionPlan: ActionPlan) {
-    runBlocking {
+  override suspend fun actionPlanCreated(actionPlan: ActionPlan) {
+    coroutineScope {
```

Comment:
Use the coroutine scope from the suspend function, rather than creating a blocking scope

Reply:
As per @mikehalmamoj's suggested approach to doing fire & forget, this is wrong, and should be the injected scope bean 👍